madrasthemes/mas-elementor
# Fix/text domain

### Steps to test the issue

1. Install Plugin Check (PCP) plugin.
2. Pull this branch  and rename plugin folder as mas-addons-for-elementor.
3. Go to Dashboard > Plugins > MAS Addons For Elementor > check this plugin and check for errors. Except performance checklist warnings, other checklists should not have error.
![image](https://github.com/user-attachments/assets/30563788-3f64-4f15-84fd-be819f858122)
